### ISSUE TITLE
Warning SC2183 incorrectly interprets format strings

### ISSUE THREAD

**Issue by KellerFuchs** [koalaman/shellcheck#987]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2183
- My shellcheck version (`shellcheck --version` or "online"): online & 0.4.5 (on Travis CI, running Ubuntu Trusty)
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/sh
printf -- ' %72s\n'
```

#### Here's what shellcheck currently says:
```
printf -- ' %72s\n'
          ^-- SC2183: This format string has 1 variables, but is passed 0 arguments.
```

#### Here's what I wanted or expected to see:

No warning.

**Comment by KellerFuchs** [comment]
For information, the issue is not present on 0.4.4 (Debian 9.1 “jessie”)

**Comment by KellerFuchs** [comment]
Nevermind, I think the correct imprecation ought to be `printf ' %72s\n' ' '`.
